src/api/status/public/pages/build.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <base href="/v1/status/" />
+    <!-- <base href="/v1/status/" /> -->
```

Comment:
Revert this. Running `yarn dev` inside /src/ap/status should be working out-of-the-box.